Samples/PushNotifications/cpp-console-unpackaged/cpp-console-unpackaged.cpp

```diff
@@ -56,7 +55,7 @@ winrt::Windows::Foundation::IAsyncOperation<PushNotificationChannel> RequestChan
             if (args.status == PushNotificationChannelStatus::InProgress)
             {
                 // This is basically a noop since it isn't really an error state
-                std::cout << "Channel request is in progress." << std::endl;
+                std::cout << std::endl << "Channel request is in progress." << std::endl;
```

Comment:
endl is equivalent to \n and flushing the buffer. It's mostly useless on a console because there are many events that will flush the buffer. The added \n, just ensures some extra breathing room so that output isn't all bunched up. It makes it easier to read.